### ISSUE TITLE
No document type declared which forces "quirks mode"

### ISSUE THREAD

**Issue by NoxHarmonium** [AustralianSynchrotron/openday-scavenger#109]
It's probably not urgent but it is best practice to put a 
`<!doctype html>` tag at the top of each HTML page to disable "quirks mode" which causes browser warnings and makes things less consistent

See: https://developer.mozilla.org/en-US/docs/Web/HTML/Quirks_Mode_and_Standards_Mode

![image](https://github.com/user-attachments/assets/94b1d661-65dd-484a-a784-e5ffd0466de2)

**Comment by NoxHarmonium** [comment]
Might be good to add a viewport tag too:

https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag

**Comment by NoxHarmonium** [comment]
Updated in https://github.com/AustralianSynchrotron/openday-scavenger/compare/sd-meta-tags?expand=1

**Comment by chanmSC** [comment]
Agreed, would be good to have everyone's `index.html` or `layout.html` have:
```
<!DOCTYPE html>
<html lang="en">

<head>
    <title>{{ puzzle name }}  | Australian Synchrotron Open Day Scavenger Hunt</title>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link id="favicon" rel="icon" type="image/x-icon" href="/static/favicon.ico">
    <link rel="manifest" href="/static/manifest.webmanifest" />
```

And add the manifest file I mention in this other issue: https://github.com/AustralianSynchrotron/openday-scavenger/issues/90

But everyone was already working on their own index pages, and a message on Teams wasn't going to get everyone updated, lol.
